tools/jscodeshift/convert-enzyme-to-rtl.js

```diff
@@ -0,0 +1,311 @@
+    if (numberOfImportsFromSource) {
+        if (numberOfImportsFromSource === 1) {
+            importDeclaration.remove();
+        } else {
+            const namedImportSpecifier = root.find(j.ImportSpecifier, {
+                imported: { name },
+            });
+            namedImportSpecifier.remove();
+        }
+    }
+}
+
+function getCallExpressionByCalleeName(j, root, name) {
+    return root.find(j.CallExpression, { callee: { name } });
+}
+
+function getVariableDeclaratorByCalleeName(j, root, name) {
+    return root.find(j.VariableDeclarator, { init: { callee: { name } } });
+}
+
+const renameVariableDeclarator = (j, path, newIdentifier) => {
+    j(path).renameTo(newIdentifier);
+};
+
+const replaceCallExpressionIdentifier = (j, root, identifier, newIdentifier, args) => {
+    getCallExpressionByCalleeName(j, root, identifier).forEach(ce => {
+        j(ce).replaceWith(
+            j.callExpression(
+                j.identifier(newIdentifier),
+                args !== undefined ? args : j(ce).get().value.arguments,
+            ),
+        );
+    });
+};
+
+const appendIfNecessary = (name, newProperty, appendage) => {
+    const appendageDefault = 'container.';
+    return `${
+        name === placeholderVariableName
+            ? typeof appendage === 'string'
+                ? appendage
+                : appendageDefault
+            : ''
+    }${newProperty}`;
+};
+
+const replaceCallExpressionWithMemberExpression = (
+    j,
+    path,
+    newProperty,
+    appendage = '.container',
+) => {
+    const callee = path.get().value.callee || path.get().value;
+    if (callee?.object?.name === placeholderVariableName) {
+        callee.object.name += appendage;
+    }
+    path.replaceWith(j.memberExpression(callee.object, newProperty));
+};
+
+const replaceCallExpressionProperty = (j, callExpression, newProperty, args, appendage) => {
+    const callee = j(callExpression).get().value.callee;
+    if (appendage !== undefined) {
+        newProperty = appendIfNecessary(callee.object.name, newProperty, appendage);
+    }
+    callee.property = j.identifier(newProperty);
+    j(callExpression).replaceWith(
+        j.callExpression(
+            callee,
+            args !== undefined ? args : j(callExpression).get().value.arguments,
+        ),
+    );
+};
+
+const getCallExpressionByProperty = (j, root, name) => {
+    return root.find(j.CallExpression, { callee: { property: { name } } });
+};
+
+function transformer(fileInfo, api) {
+    const j = api.jscodeshift;
+    const root = j(fileInfo.source);
+    // FIND ENZYME IMPORT
+    const enzymeImportDeclaration = root.find(j.ImportDeclaration, { source: { value: 'enzyme' } });
+    if (enzymeImportDeclaration.length === 0) {
+        return;
+    }
+
+    const getFirstNode = () => root.find(j.Program).get('body', 0).node;
+
+    // Save the comments attached to the first node then remove them
```

Comment:
Comments in this file are sort of inconsistent--is it worth having a consistent style? (definitely not a blocker)

Line 110: `// FIND...`
Line 123: `// add...`
Line 126: `//remove...`
Line 133: `///remove...`